[2023-04-08T10:16:44.895+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-08T07:00:00+00:00 [queued]>
[2023-04-08T10:16:44.994+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-08T07:00:00+00:00 [queued]>
[2023-04-08T10:16:45.001+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T10:16:45.004+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-08T10:16:45.008+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T10:16:45.095+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-08 07:00:00+00:00
[2023-04-08T10:16:45.116+0000] {standard_task_runner.py:55} INFO - Started process 13699 to run task
[2023-04-08T10:16:45.151+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'scheduled__2023-04-08T07:00:00+00:00', '--job-id', '315', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmp28ph1dcw']
[2023-04-08T10:16:45.176+0000] {standard_task_runner.py:83} INFO - Job 315: Subtask respond
[2023-04-08T10:16:45.547+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond scheduled__2023-04-08T07:00:00+00:00 [running]> on host cfece00ff772
[2023-04-08T10:16:45.828+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-08T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-08T07:00:00+00:00
[2023-04-08T10:16:45.855+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-08T10:16:45.969+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230408T070000, start_date=20230408T101644, end_date=20230408T101645
[2023-04-08T10:16:46.087+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-08T10:16:46.229+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
